From: SeongJae Park <sj@kernel.org>
Date: Mon, 20 Jan 2025 13:43:38 -0800
Subject: [PATCH] ==== auto-tune monitoring parameters ====

mm/damon: auto-tune monitoring intervals aiming an amount of accesses

Why aggregation interval is important and need tuning.

Why aggregation interval tuning is challenging.

Why the amount of observed memory access is a better metric.

How the auto-tuning is designed.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_448v7nrp | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_448v7nrp

diff --git a/hkml_cv_bogus/hkml_cv_bogus_448v7nrp b/hkml_cv_bogus/hkml_cv_bogus_448v7nrp
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

